tests/OSPSuite.Core.Tests/Core/FavoritesSpecs.cs

```diff
@@ -0,0 +1,113 @@
+﻿using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain;
+
+namespace OSPSuite.Core
+{
+   public abstract class concern_for_Favorites : ContextSpecification<Favorites>
```

Comment:
Pure BDD implementation here. Defined the tests first and was done when they were all green :)